From e445cc94831c55b6c5430716bc03613b7a2bb83b Mon Sep 17 00:00:00 2001
From: Petr Vorel <pvorel@suse.cz>
Date: Fri, 10 Mar 2023 11:39:00 +0100
Subject: [PATCH] fsverity.test: Add /usr/sbin into $PATH

Unlike GitHub CI, tests which run by packaging tools often run as non-root,
thus required tools aren't visible:

./functions.sh: line 45: type: mkfs: not found
mkfs is required for test
./functions.sh: line 45: type: blkid: not found
blkid is required for test
./functions.sh: line 45: type: e2fsck: not found
e2fsck is required for test
./functions.sh: line 45: type: tune2fs: not found
tune2fs is required for test

Signed-off-by: Petr Vorel <pvorel@suse.cz>
---
Upstream status: https://lore.kernel.org/linux-integrity/20230310104729.32078-1-pvorel@suse.cz/

 tests/fsverity.test | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/fsverity.test b/tests/fsverity.test
index 01d5c35..a274158 100755
--- a/tests/fsverity.test
+++ b/tests/fsverity.test
@@ -30,7 +30,7 @@
 # custom policy rules might take precedence.
 
 cd "$(dirname "$0")" || exit 1
-PATH=../src:../fsverity-utils:$PATH
+PATH=../src:../fsverity-utils:/usr/sbin:$PATH
 source ./functions.sh
 
 # Base VERBOSE on the environment variable, if set.
-- 
2.39.2

